<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="processButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработать</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Верхний регистр</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Нижний регистр</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инвертирование</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Удаление пробелов</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
